packages/dashboard/src/components/DashboardGridLayout/DashboardGridLayoutBase.tsx

```diff
@@ -38,7 +37,15 @@ export class DashboardGridLayoutBase extends React.Component<IDashboardGridLayou
       sm: 1024,
       xs: 640,
       xxs: 480
-    }
+    },
+    cardSizeToRGLWidthHeight: {
```

Comment:
What's the reason for exposing this as a prop?

Reply:
Since we were working with spacing and sizing issue this morning, I feel that we might need to have the flexibility cardsize to RGL size mapping

---
In reply to: [217188173](https://github.com/OfficeDev/office-ui-fabric-react/pull/6341#discussion_r217188173) [](ancestors = 217188173)